lib/dal/src/attribute/prototype/argument.rs

```diff
@@ -543,17 +544,47 @@ impl AttributePrototypeArgument {
         Ok(apas)
     }
 
+    /// Removes the [`AttributePrototypeArgument`] corresponding to the provided ID.
     pub async fn remove(
         ctx: &DalContext,
-        apa_id: AttributePrototypeArgumentId,
+        attribute_prototype_argument_id: AttributePrototypeArgumentId,
     ) -> AttributePrototypeArgumentResult<()> {
-        let apa = Self::get_by_id(ctx, apa_id).await?;
-        let prototype_id = apa.prototype_id(ctx).await?;
+        let attribute_prototype_argument =
+            Self::get_by_id(ctx, attribute_prototype_argument_id).await?;
+        attribute_prototype_argument.remove_inner(ctx).await?;
+        Ok(())
+    }
+
+    /// Removes the [`AttributePrototypeArgument`] corresponding to the provided ID, but is a
+    /// "no-op" if it cannot be found before removal.
+    pub async fn remove_or_no_op(
```

Comment:
Did you get this from #3578? Curious to why the diff would show this

Reply:
I actually don't know what happened here... let's chat